db/migrate/20170329230309_create_requested_topics.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRequestedTopics < ActiveRecord::Migration[5.0]
+  def change
+    create_table :requested_topics do |t|
+      t.string :name, null: false
```

Comment:
I wanted to go with `.topic` but I threw it around the pod and `.name` came out. It's tricky because we are naming a presentations topic prop as 'topic' so topic is kind of what I expect this thing to be named as. If we have a TopicRequest I think name and my suggestion of `topic_request.topic` are the only ones that really make sense to me. I think `topic_request.topic` kind of sucks because it's a prop name named after the class.